### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by aeronjl**
I fleshed this out to make the prose more conversational and hopefully make some points clearer.

### CODE REVIEW

File: birkbeck/db1/README.md [hunk 1]
```diff
@@ -2,63 +2,76 @@
...
+Command-line interfaces respond to commands. These are short words which may sometimes be followed by a file path or optional characters to specify additional details. Here are a few of the most popular and useful command-line commands:
 
-![a screen terminal](https://upload.wikimedia.org/wikipedia/commons/8/87/Televideo925Terminal.jpg)
+- `whoami`
+No, this isn't a philosophy lesson. This command simply returns the username you are currently using to access the system. This can be useful to check you have logged in correctly.
 
-The most used and useful commands for CLI are:
-- `whoami` (sounds philosophical but it is helpful)
 - `pwd` (Print Working Directory)
+In other words, where am I? This command tells you which directory (or folder) you are currently operating in. This is important to know, because actions taken which do not specify an explicit alternate directory will result in changes being made to the directory you are currently in. Use this to make sure you're in the right place when deleting old files, and not accidentally deleting the contents of your coursework folder.
```
Comment (wizofe) [inline birkbeck/db1/README.md]: I find this a bit wordy. I wanted to keep the Unix intro as short as possible, so I would say to only keep the first two sentences; i.e. until "...operating in.".

File: birkbeck/db1/README.md [hunk 2]
```diff
@@ -2,63 +2,76 @@
 
 ## A quick intro to `ssh`
 
-`SSH` -as its name implies- is a secure shell. It creates a strongly encrypted connection between a local (your device) and a remote server (a machine you cannot usually physically access.)
+`SSH` stands for Secure Shell, a cryptographic protocol used to connect devices over a potentially unsecured network connection. It has been in use since the late 1990s primarily as a way to access remote servers requiring user authentication, which is exactly what we are trying to do by connecting remotely to the `hope` server.
 
-In our case, we need to access the server of Birkbeck Biology Department (a.k.a. hope) using our device (a laptop, desktop, et.c.)
+Using `SSH` successfully does not require advanced knowledge of network security or mathematical encryption. However, it does involve navigating a UNIX-like command-line interface. This tutorial will take you through all the basics you need to know to get started with the command line on your operating system and connect to `hope` with minimum struggle.
```
Comment (wizofe) [inline birkbeck/db1/README.md:7]: That's great!

**Review by wizofe — CHANGES_REQUESTED** [review]
Thank you for the PR, that's great! My only objection is regarding the lengthy text about `pwd`. ta

File: birkbeck/db1/README.md (hunk 1 above)
Comment (aeronjl) [reply birkbeck/db1/README.md]: Yep, fair!

**Comment by aeronjl** [comment]
I made the change you asked for. 👍

**Review by wizofe — APPROVED** [review]

**Comment by wizofe** [comment]
Adjustments and prose beautification from @aeronlaffere!
